src/tools/FreehandMouseTool.js

```diff
@@ -501,20 +501,25 @@ export default class FreehandMouseTool extends BaseAnnotationTool {
 
     const config = this.configuration;
 
-    const nearby = this._pointNearHandleAllTools(eventData);
-    const handleNearby = nearby.handleNearby;
-    const toolIndex = nearby.toolIndex;
-
+    //const nearby = this._pointNearHandleAllTools(eventData);
```

Comment:
What is the rationale for replicating _pointNearHandleAllTools() with a loop doing the same thing? It was factored into functions to make this call easier to follow complicated. Happy to discuss this point if you think there is a valid reason to change this.

Reply:
_pointNearHandleAllTools, by calling _pointNearHandle, checks if the point is less than a distance of config.spacing (=5 for default) away from the handle. When we call handleSelectedCallback we already know that a handle was selected. In touchStart.js the distance must be less than distanceFromHandle (=28 for default) for the handleSelectedCallback to be called.

Reply:
maybe config.spacing has to be changed anyway when using touch, as the value of 5 makes it hard to complete a polygon